### PR TITLE
chore: release 0.31.0

### PR DESCRIPTION
**Pull request by Milouu**
- Add user management doc ([#345](https://github.com/Substra/substra-documentation/pull/345))
- Add link to python libraries documentation in Components page ([#347](https://github.com/Substra/substra-documentation/pull/347))
- Add more orchestrator documentation ([#346](https://github.com/Substra/substra-documentation/pull/346))
- Deactivate Binder ([#340](https://github.com/Substra/substra-documentation/pull/340))
- Reorganise documentation according to [diataxis](https://diataxis.fr/) approach ([#330](https://github.com/Substra/substra-documentation/pull/330))
- Added section about channels in the main concepts ([#344](https://github.com/Substra/substra-documentation/pull/344))
- Add frontend documentation in components section ([#346](https://github.com/Substra/substra-documentation/pull/346))
- New page added on Substra Privacy Strategy based on research by Privacy Task Force at Owkin ([#354](https://github.com/Substra/substra-documentation/pull/354))
- Revamp landing page ([#353](https://github.com/Substra/substra-documentation/pull/353))

### CODE REVIEW

File: docs/source/additional/releases.yaml [hunk 1]
```diff
@@ -9,11 +9,49 @@ components: # for table headers
...
+      substra-tests:
+        version: 0.44.0
+        link: https://github.com/Substra/substra-tests/releases/tag/0.44.0
   - version: 0.30.0
     components:
       substrafl:
-        version: 0.39.0
-        link: https://github.com/Substra/substrafl/releases/tag/0.39.0
+        version: 0.39.1
+        link: https://github.com/Substra/substrafl/releases/tag/0.39.1
```
Comment (oleobal) [inline docs/source/additional/releases.yaml:54]: I don't think we should be changing what existing version numbers mean. If we want to integrate these patches let's designate it `0.30.1` -- but really I think a `0.31.0` is sufficient, who would upgrade to `0.30.1` but not `0.31.0`?

**Review by oleobal — CHANGES_REQUESTED** [review]

File: docs/source/additional/releases.yaml (hunk 1 above)
Comment (oleobal) [reply docs/source/additional/releases.yaml:54]: But that's rewriting history. We're not going to tell people "Oh, are you using the _old_ 0.30.0 or the _new_ 0.30.0?" -- getting rid of this confusion is the point of using version numbers. IMO let's just not bother, but I suppose we could also release a 0.30.1.

Comment (Milouu) [reply docs/source/additional/releases.yaml:54]: That's true but should also have 0.29.1 & 0.28.1 then and a bunch other fixes which we never officially made a bundle release for. Let's not bother with this debate rn, i'll put it back to what i did first.

Comment (ThibaultFy) [reply docs/source/additional/releases.yaml]: Mmmmh I get what you mean @oleobal and you're right... can't wait to the merge of all these repo to have ONE version for all...

**Review by ThibaultFy — APPROVED** [review]

**Pull request merged**